bandit/plugins/tarfile_unsafe_members.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2019 VMware, Inc.
```

Comment:
Please remove this line.  Intended for VMware employees.  However, feel free to add your copyright.

Reply:
```suggestion
```